client/src/mocks/handlers.ts

```diff
@@ -6,4 +6,30 @@ export const handlers = [
   rest.get('/cards', (req, res, ctx) => {
     return res(ctx.status(200), ctx.json(cafes));
   }),
+
+  // 좋아요 추가
+  rest.post('/cafes/:cafe_id/likes', (req, res, ctx) => {
+    // eslint-disable-next-line camelcase
+    const { cafe_id } = req.params;
+    const updatedCafes = cafes.map((cafe) => {
+      if (cafe.id === Number(cafe_id)) {
+        return { ...cafe, likes: cafe.likeCount + 1 };
+      }
+      return cafe;
+    });
+    return res(ctx.status(200), ctx.json(updatedCafes));
+  }),
```

Comment:
```suggestion
  rest.post('/cafes/:cafeId/likes', (req, res, ctx) => {
    const { cafeId } = req.params;
    const updatedCafes = cafes.map((cafe) => {
      if (cafe.id === Number(cafeId)) {
        return { ...cafe, likes: cafe.likeCount + 1 };
      }
      return cafe;
    });
    return res(ctx.status(200), ctx.json(updatedCafes));
  }),
```
`cafe_id` 를 camelCase로 작성하면 좋지 않을까요?!

Reply:
뭔가 API 이름을 맞춰야 해야하나 해서 그렇게 했는데 저희 컨벤션 상 생각해보면 camelCase가 더 올바른 거 같네여

Reply:
params의 이름은 안 맞춰줘도 될 것 같아요~ 어차피 placeholder니까요